src/compiler/checker.ts

```diff
@@ -30004,7 +30004,9 @@ namespace ts {
         function checkImportBinding(node: ImportEqualsDeclaration | ImportClause | NamespaceImport | ImportSpecifier) {
             checkCollisionWithRequireExportsInGeneratedCode(node, node.name!);
             checkCollisionWithGlobalPromiseInGeneratedCode(node, node.name!);
-            checkAliasSymbol(node);
+            if (!isInJSFile(node) || languageVersion < ScriptTarget.ES2020) {

```

Comment:
I think you want to skip reporting error and not checking the symbol.

Reply:
I see, move it closer to the error emission 👍